bin/build_zip_and_copy_to_infra.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+# e.g. ~/workshop/udx-infra/terraform/modules/data-lake
```

Comment:
Do we need this script anymore if we aren't storing the build in the infra repo?

Reply:
Yep, this is entirely superflous